wiki/Gameplay/Replay/fr.md

```diff
@@ -18,7 +18,7 @@ Pour relancer un replay depuis le début, restez appuyé sur un des boutons par
 L'écran devrait s'assombrir et un son sera joué lorsque le replay sera relancé.
 Relâcher trop tôt ne permettra pas au relancement de s'effectuer.
 
-Il est toujours possible d'accéder à la [console de chat](/wiki/Client/Interface/Chat_console) (`F8`)/[console de chat étendue](/wiki/Client/Interface/Chat_console#la-console-de-chat-étendue) (`F9`) .
+Il est toujours possible d'accéder à la [console de tchat](/wiki/Client/Interface/Chat_console) (`F8`)/[console de tchat étendue](/wiki/Client/Interface/Chat_console#la-console-de-tchat-étendue) (`F9`) .
```

Comment:
you've changed this link but haven't changed the heading in the chat console article